lib/pronto/formatter/git_formatter.rb

```diff
@@ -10,7 +10,7 @@ def format(messages, repo, patches)
         
         approve_pull_request(comments.count, additions.count, client) if defined?(self.approve_pull_request)
 
-        "#{additions.count} Pronto messages posted to #{pretty_name}"
+        "New #{additions.count} (to #{comments.count} existed) Pronto messages posted to #{pretty_name}"
```

Comment:
```suggestion
        "#{additions.count} Pronto messages posted to #{pretty_name} (#{comments.count} existing)"
```
I'd prefer the additional information was at the end.